### PR TITLE
Small tweaks for better speed

### PR DESCRIPTION
**Pull request by liviuba**
For this https://app.clickup.com/t/8696kk2bj and this https://app.clickup.com/t/8696kmfu6

### CODE REVIEW

File: api/api/settings.py [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+from pydantic import Field
 from pathlib import Path
 
 
 class Settings(BaseSettings):
     mongo_connstring: str
     mongo_timeout_ms: int
+    mongo_max_pool_size: int = Field(
+        default=10,
+        description="Max size of the Mongo connection pool. If 1 process/api, this should be higher if the link to Mongo is fast",
```
Comment (kbab) [inline api/api/settings.py:11]: I am not clear what the statement "If 1 process/api," means.

Comment (liviuba) [reply api/api/settings.py]: updated the text now

**Comment by liviuba** [comment]
Just noticed tests fail, it's because of the on_start changes, I'll fix that later, it's ok to ignore for now

**Review by kbab — APPROVED** [review]
LGTM (apart from tests issue)

**Pull request merged**